faqs/atm7_http.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How to enable HTTP on >1.12
+search: atm7 http
+---
+
+**Singleplayer**
+1. In CurseForge, go to "My Modpacks" and right-click the ATM-7 modpack.
```

Comment:
Should we really assume that person is using curseforge to play this modpack? This FAQ really should be more generic not specific to launcher.

Reply:
Chummy decided on that, what would you prefer?

Reply:
Considering title is generic "how to turn on http" we really should not assume anything about operating system, launcher or modpack. After all this might be used for many other uses then just AMT7

Reply:
Well how do i include all of them